information.css

```diff
@@ -0,0 +1,21 @@
+#infor
+{
+    border: 1px #F2F2F2 solid;
+    width: 19px;
+    border-radius: 40px;
+    text-align: center;
+    color: #0065FC;
+    margin-left: 15px;
+    margin-top: 24px;
+}
+
+#inform
+{
+    display: flex;
+}
+
+#informa
```

Comment:
inform, informa, infor... why are you doing that to yourself ! it's seem's you have a lot of class with the same purpose ... or a wrong naming in my opinion.

Reply:
Changed !